docs/source/deployment/deploy-model-to-kubernetes/tutorial.rst

```diff
@@ -0,0 +1,484 @@
+
+        If you already have access to a Kubernetes cluster, you can install KServe to your cluster by following `the official instructions <https://github.com/kserve/kserve#hammer_and_wrench-installation>`_.
+
+  .. tab:: Local Machine Emulation
+
+        You can follow `KServe QuickStart <https://kserve.github.io/website/latest/get_started/>`_ to set up a local cluster with `Kind <https://kind.sigs.k8s.io/docs/user/quick-start>`_
+        and install KServe on it.
+
+Now that you have a Kubernetes cluster running as a deployment target, let's move on to creating the MLflow Model to deploy.
+
+Step 3: Training the Model
+--------------------------
+
+In this tutorial, we will train and deploy a simple regression model that predicts the quality of wine.
+
+Let's start from training a model with the default hyperparameters. Execute the following code in a notebook or as a Python script.
+
+.. note::
+
+  For the sake of convenience, we use the `mlflow.sklearn.autolog() <../../../python_api/mlflow.sklearn.html#mlflow.sklearn.autolog>`_ function.
+  This function let MLflow to automatically log the appropriate set of model parameters and metrics. To learn more about the autologging feature
+  or how to log manually instead, see the `MLflow Tracking documentation <../../../tracking.html>`_.
+
+.. code-block:: python
+
+  import mlflow
+
+  from sklearn import datasets
+  from sklearn.linear_model import ElasticNet
+  from sklearn.model_selection import train_test_split
+
+
+  def eval_metrics(pred, actual):
+      rmse = np.sqrt(mean_squared_error(actual, pred))
+      mae = mean_absolute_error(actual, pred)
+      r2 = r2_score(actual, pred)
+      return rmse, mae, r2
+
+
+  # Set th experiment name
+  mlflow.set_experiment("wine-quality")
+
+  # Enable auto-logging to MLflow
+  mlflow.sklearn.autolog()
+
+  # Load wine quality dataset
+  X, y = datasets.load_wine(return_X_y=True)
+  X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.25)
+
+  # Start a run and train a model
+  with mlflow.start_run(run_name="default-params"):
+      lr = ElasticNet()
+      lr.fit(X_train, y_train)
+
+      y_pred = lr.predict(X_test)
+      metrics = eval_metrics(y_pred, y_test)
+
+
+Now you have trained a model, let's check if the parameters and metrics are logged correctly, via the MLflow UI.
+You can start the MLflow UI by running the following command in your terminal:
+
+.. code-block:: bash
+
+  mlflow ui --port 5000
+
+Then visit http://localhost:5000 to open the UI.
+
+.. figure:: ../../_static/images/deployment/tracking-ui-default.png
+    :align: center
+    :figwidth: 80%
+
+Please open the experient named "wine-quality" on the left, then click the run named "default-params" in the table.
+For this case, you should see parameters including ``alpha`` and ``l1_ratio`` and metrics like ``training_score`` and ``mean_absolute_error_X_test``.
+
+Step 4: Running Hyperparameter Tuning
+-------------------------------------
+
```

Comment:
We might want to add a ref to https://www.mlflow.org/docs/latest/traditional-ml/hyperparameter-tuning-with-child-runs/notebooks/hyperparameter-tuning-with-child-runs.html 
For people who want to see a more in-depth guide on hyperparameter tuning

Reply:
Great idea, will add!